src/nectarchain/user_scripts/dmousadi/test_scripts/tests/utils.py

```diff
@@ -0,0 +1,500 @@
+        b (float): The coefficient of the linear term.
+        c (float): The constant term.
+    
+    Returns:
+        float: The result of the quadratic function.
+    """
+    return a*(x**2) + b*x + c
+
+def fit_function3(x,a,b,c, d):
+    """
+    Computes a function of the form `(a*x + b)/(1+c) + d`.
+    
+    Args:
+        x (float): The input value.
+        a (float): The coefficient of the linear term.
+        b (float): The constant term in the numerator.
+        c (float): The coefficient of the denominator.
+        d (float): The constant term added to the result.
+    
+    Returns:
+        float: The result of the function.
+    """
+    return (a*x + b)/(1+c) + d
+
+def fit_function_hv(x,a,b):
+    """
+    Computes a function of the form `a/sqrt(x) + b`.
+    
+    Args:
+        x (float): The input value.
+        a (float): The coefficient of the term `1/sqrt(x)`.
+        b (float): The constant term.
+    
+    Returns:
+        float: The result of the function.
+    """
+    return a/np.sqrt(x)+b
+
+
+def err_ratio(nominator, denominator, err_norm, err_denom, cov_nom_den = 0):
+    """
+    Computes the error ratio for a given nominator, denominator, and their respective errors.
+    
+    Args:
+        nominator (float): The nominator value.
+        denominator (float): The denominator value.
+        err_norm (float): The error of the nominator.
+        err_denom (float): The error of the denominator.
+        cov_nom_den (float, optional): The covariance between the nominator and denominator. Defaults to 0.
+    
+    Returns:
+        float: The error ratio.
+    """
+    delta_err2 = (err_norm/nominator)**2 + (err_denom/denominator)**2 -2*cov_nom_den/(nominator*denominator)
+    ratio = nominator/denominator
+    return np.sqrt(delta_err2)*ratio
+
+def err_sum(err_a, err_b, cov_a_b=0):
+    """
+    Computes the square root of the sum of the squares of `err_a` and `err_b`, plus twice the covariance `cov_a_b`.
+    
+    This function is used to calculate the combined error of two values, taking into account their individual errors and the covariance between them.
+    
+    Args:
+        err_a (float): The error of the first value.
+        err_b (float): The error of the second value.
+        cov_a_b (float, optional): The covariance between the two values. Defaults to 0.
+    
+    Returns:
+        float: The combined error.
+    """
+    return np.sqrt(err_a**2 + err_b**2 + 2*cov_a_b)
+
+def closest_value(input_list, input_value):
+    """
+    Returns the value in `input_list` that is closest to `input_value`.
+    
+    Args:
+        input_list (list): The list of values to search.
+        input_value (float): The value to find the closest match for.
+    
+    Returns:
+        float: The value in `input_list` that is closest to `input_value`.
+    """
+    arr = np.asarray(input_list)
+    i = (np.abs(arr - input_value)).argmin()
+    return arr[i]
+
+def gaus(x,a,mu,sigma):
```

Comment:
I guess just using a function from `numpy` or `scipy` would do, instead of re-implementing it here.